cypher-shell/src/main/java/org/neo4j/shell/state/BoltStateHandler.java

```diff
@@ -105,11 +97,7 @@ public void connect(@Nonnull ConnectionConfig connectionConfig) throws CommandEx
             // Bug in Java driver forces us to run a statement to make it actually connect
             session.run("RETURN 1").consume();
         } catch (Throwable t) {
-            try {
-                silentDisconnect();
-            } catch (Exception e) {// NOPMD
-                // This is to ensure we are able to show the original message by exception to to the user
-            }
+            silentDisconnect();
```

Comment:
As a minimum, instead of the empty `catch`, you might want to consider `t.addSuppressed(e)`.